lib/features/digital_guide_view/presentation/widgets/digital_guide_features_section.dart

```diff
@@ -0,0 +1,74 @@
+import "package:flutter/material.dart";
+import "package:flutter/widgets.dart";
+
+import "../../../../config/ui_config.dart";
+import "../../../../utils/context_extensions.dart";
+import "../../../../widgets/my_expansion_tile.dart";
+import "expansion_tiles_content/localization_expansion_tile_content.dart";
+
+class DigitalGuideFeaturesSection extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 24, right: 24, top: 24, bottom: 8),
+      child: Column(
```

Comment:
I don't like the fact that this is one Column. In case of such long lists this can cause performance problems while scrolling on not top tier devices.

Reply:
@simon-the-shark Do you have any idea what could I use instead?

Reply:
@24bartixx  you want anything builder  based (`ListView.builder` or anything similar), not children based. You have 2 options here:
1. You don't extract this column here, but place all widgets in the CustomScrollableBuilder with BuilderDelegate (the widget more to the top of the tree). Then you have one scrollable, it's builder based and works smoothly, however it's not extarcted to separate widget nor file.

```dart
[
// ...
DigitalGuideFeaturesSection(digitalGuideResponseExtended: digitalGuideResponseExtended), // either just frop the Columns content here (but it will get lengthy and nasty)
ReportChangeButton(),
//...
]
```
3. If you want to preserve this extraction like you did here, you can still change the `Column` to `ListView.builder`, but for it to work inside other scrollable without crashing you need to disable its scrollability and expandibility. In this situation this seems like a better idea.
```dart
import "package:flutter/material.dart";
import "package:flutter/widgets.dart";

import "../../../../../config/ui_config.dart";
import "../../../../../utils/context_extensions.dart";
import "../../../../../widgets/my_expansion_tile.dart";
import "../../../amenities/presentation/amenities_expansion_tile_content.dart";
import "../../../localization/presentation/localization_expansion_tile_content.dart";
import "../../../surrounding/presentation/surroundings_expansion_tile_content.dart";
import "../../data/models/digital_guide_response_extended.dart";

class DigitalGuideFeaturesSection extends StatelessWidget {
  const DigitalGuideFeaturesSection({
    required this.digitalGuideResponseExtended,
  });

  final DigitalGuideResponseExtended digitalGuideResponseExtended;

  @override
  Widget build(BuildContext context) {
    final widgets = [
      MyExpansionTile(
        title: context.localize.localization,
        children: [LocalizationExpansionTileContent()],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      MyExpansionTile(
        title: context.localize.amenities,
        children: [
          AmenitiesExpansionTileContent(
            digitalGuideResponseExtended: digitalGuideResponseExtended,
          ),
        ],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      MyExpansionTile(
        title: context.localize.surroundings,
        children: [
          SurroundingsExpansionTileContent(
            digitalGuideResponseExtended: digitalGuideResponseExtended,
          ),
        ],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      MyExpansionTile(
        title: context.localize.transport,
        children: [LocalizationExpansionTileContent()],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      MyExpansionTile(
        title: context.localize.entrances,
        children: [LocalizationExpansionTileContent()],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      MyExpansionTile(
        title: context.localize.elevators,
        children: [LocalizationExpansionTileContent()],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      // there is no data provided in any endpoint for this expansion tile (toilets)
      // This tile is not implemented on digital guide website XDDDD
      // Can I skip it?
      MyExpansionTile(
        title: context.localize.toilets,
        children: [LocalizationExpansionTileContent()],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      MyExpansionTile(
        title: context.localize.micro_navigation,
        children: [LocalizationExpansionTileContent()],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      MyExpansionTile(
        title: context.localize.building_structure,
        children: [LocalizationExpansionTileContent()],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      MyExpansionTile(
        title: context.localize.room_information,
        children: [LocalizationExpansionTileContent()],
      ),
      const SizedBox(height: DigitalGuideConfig.heightMedium),
      MyExpansionTile(
        title: context.localize.evacuation,
        children: [LocalizationExpansionTileContent()],
      ),
    ];
    return Padding(
      padding: const EdgeInsets.only(left: 24, right: 24, top: 24, bottom: 8),
      child: ListView.builder(
        itemBuilder: (context, index) {
          return widgets[index];
        },
        itemCount: widgets.length,
        shrinkWrap:
            true, // thanks to this this will work inside other scrollable too
        physics:
            const NeverScrollableScrollPhysics(), // thanks to this this will work inside other scrollable too
      ),
    );
  }
}
```